config/collectdata/python/write_organ_tags.py

```diff
@@ -36,29 +51,29 @@
 queryOutput = []
 for line in ret["results"]["bindings"]:
     queryOutput.append(line)
-# generate list of IRIs and organ labels
+
+# generate list of organ cell DL queries and semantic labels
 organs = []
 for n in queryOutput:
     IRI = n['x']['value'].replace("_", ":")
```

Comment:
The var name IRI is confusing here.

Reply:
Good catch. Please see updated var names.